raisinbread/RB_files/RB_ConfigSettings.sql

```diff
@@ -96,5 +96,6 @@ INSERT INTO `ConfigSettings` (`ID`, `Name`, `Description`, `Visible`, `AllowMult
 INSERT INTO `ConfigSettings` (`ID`, `Name`, `Description`, `Visible`, `AllowMultiple`, `DataType`, `Parent`, `Label`, `OrderNumber`) VALUES (97,'publication_uploads','Path to uploaded publications',1,0,'text',26,'Publications',11);
 INSERT INTO `ConfigSettings` (`ID`, `Name`, `Description`, `Visible`, `AllowMultiple`, `DataType`, `Parent`, `Label`, `OrderNumber`) VALUES (98,'publication_deletions','Path to deleted publications',1,0,'text',26,'Deleted Publications',11);
 INSERT INTO `ConfigSettings` (`ID`, `Name`, `Description`, `Visible`, `AllowMultiple`, `DataType`, `Parent`, `Label`, `OrderNumber`) VALUES (99,'usePwnedPasswordsAPI','Whether to query the Have I Been Pwned password API on password changes to prevent the usage of common and breached passwords',1,0,'boolean',1,'Enable \"Pwned Password\" check',22);
+INSERT INTO `ConfigSettings` (`ID`, `Name`, `Description`, `Visible`, `AllowMultiple`, `DataType`, `Parent`, `Label`, `OrderNumber`) VALUES (100,'EnvironmentFile','Name of the environment file that need to be sourced for the imaging pipeline',1,0,'text',69,'Name of the environment file',20);
```

Comment:
This ideally should not be ConfigSetting: ID=`96` in RaisinBread. It should be right after the `MRIConfigFile` setting.  We'll have a hard time checking later if all the proper config settings are in place if they're out of order with the default schema.

Reply:
RB data is handled like any project. patches are run on it and the data is submitted I never handle the data manually and if tomorrow someone decides to reorder the config settings in the 0000 files then they will go out of sync with any project out there.

If you would like to make a patch to make sure they are in sync then it can go in the New patches directory and can be run on all projects at release... I dont mind that.

Reply:
I don't feel strongly about this one way or another but I think this change request is the only point stopping this PR right now.. @christinerogers can you respond to @ridz1208 or dismiss the review?

Reply:
The issue is that the 20.2 upgrade patch was incorrectly run on RB, which caused this to happen. 
That's why we should fix it, totally understanding that other projects out there will have variance anyway from the default schema. 
I'm not in a position to make a patch so... will dismiss.